packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -246,7 +270,7 @@ contract TaikoL1 is EssentialContract, ITaikoL1 {
             blockMaxGasLimit: 240_000_000,
             livenessBond: 125e18, // 125 Taiko token
             stateRootSyncInternal: 16,
-            checkEOAForCalldataDA: true
+            ontakeForkHeight: 324_512 * 2
```

Comment:
Why not a constant ?

Reply:
Can!

Reply:
Changed to 500000 which is ~100 days away from now.